mlflow/utils/env_pack.py

```diff
@@ -0,0 +1,105 @@
+import shutil
+import subprocess
+import sys
+import tarfile
+import tempfile
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Literal, Generator
+
+from mlflow.artifacts import download_artifacts
+from mlflow.utils.databricks_utils import DatabricksRuntimeVersion
+from mlflow.utils.environment import _REQUIREMENTS_FILE_NAME
+from mlflow.utils.logging_utils import eprint
+
+EnvPackType = Literal["databricks_model_serving"]
+
+_ARTIFACT_PATH = "_databricks"
+_MODEL_VERSION_TAR = "model_version.tar"
+_MODEL_ENVIRONMENT_TAR = "model_environment.tar"
+_SUPPORTED_CLIENT_IMAGE_MAJOR_VERSIONS_FOR_MODEL_SERVING = [2, 3]
```

Comment:
```suggestion
_SUPPORTED_CLIENT_IMAGE_MAJOR_VERSIONS_FOR_MODEL_SERVING = (2, 3)
```

nit: this doesn't need to be mutable.